pensjon-brevbaker-api-model/src/main/kotlin/no/nav/pensjon/brev/api/model/maler/redigerbar/VedtakOmFjerningAvOmsorgspoengDto.kt

```diff
@@ -0,0 +1,19 @@
+package no.nav.pensjon.brev.api.model.maler.redigerbar
+
+import no.nav.pensjon.brev.api.model.maler.BrevbakerBrevdata
+import no.nav.pensjon.brev.api.model.maler.RedigerbarBrevdata
+import no.nav.pensjon.brev.api.model.vedlegg.DineRettigheterOgMulighetTilAaKlageDto
+
+@Suppress("unused")
+data class VedtakOmFjerningAvOmsorgspoengDto(
+    override val saksbehandlerValg: SaksbehandlerValg,
+    override val pesysData: PesysData,
+) : RedigerbarBrevdata<VedtakOmFjerningAvOmsorgspoengDto.SaksbehandlerValg, VedtakOmFjerningAvOmsorgspoengDto.PesysData> {
+    data class SaksbehandlerValg(
+        val aktuelleAar: String,
```

Comment:
Hva er `aktuelleAar` for noe?

Reply:
Ah, det er for å komme seg rundt at titler ikke kan redigeres enda. Kanskje få på plass en TODO-kommentar eller noe slik at vi kan plukke opp brev som må fikses på når vi får på plass redigering av titler?

Reply:
Delvis derfor, men også fordi denne verdien brukast både i tittel og fleire gongar i teksten, så dette er også ein måte å få samme verdi på alle tre-fire plassane denne brukast.